stack.yaml

```diff
@@ -131,3 +142,11 @@ ghc-options:
   cardano-sl:            -Werror
   cardano-sl-tools:      -Werror
   cardano-sl-lwallet:    -Werror
+  cardano-sl-explorer:   -Werror
+
+# Added for explorer
+flags:
+  cardano-sl:
+    with-explorer: True
```

Comment:
@gromakovsky I was talking about using `stack build ...` as a way to build the project. Can you try using that? Without that, I can't use `ghcid` and no `stack ghci`. I'm not sure about other things like editor plugins/build tools and such, but could very well be affected. 
_And that's because using `stack build ...` we implicitly include _explorer_ since it's in `packages`._

I also tried setting `with-explorer` to `False`, used `./scripts/build/cardano-sl.sh sl`. That went fine and compiled. That's not the issue. The issue is **building both projects** without setting the flag to true.

How are we going to build `explorer` with that flag set to `False`? And I don't mean the b. part of the `explorer`. I mean the c. part of the `explorer`.